modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorHelper.java

```diff
@@ -140,17 +140,9 @@ public DDMFormEvaluatorEvaluateResponse evaluate() {
 			ddmFormRules = _ddmForm.getDDMFormRules();
 		}
 
-		Stream<DDMFormRule> stream = ddmFormRules.stream();
+		_evaluateDDMFormRules(ddmFormRules, false);
```

Comment:
@rijema, what is the purpose of evaluate a rule with condition equals to `false`?

Reply:
Sending this boolean value in the signature of the method allows me to compare with the current value of evaluateDDMFormRuleCondition(ddmFormRule.getCondition(). This filter will separate the fields whose are affected by a rule and field that aren't. False means that I want to evaluate on the else condition of _evaluateDDMFormRule the fields that aren't affect, and True means that I want to evaluate on the if statement of _evaluateDDMFormRule the rules that are affecting the forms. Filtering the falses before the trues helps me to organize this separation, allowing me to keep the changes in forms (generated by the rules that are acting on the forms) and ensuring that they are not overwritten by rules that are no longer affecting. Did it help? I'm available for further explanation.